packages/table/preview/store.ts

```diff
@@ -13,25 +15,28 @@ class GridEntry<T> {
     public constructor (
         public i: number,
         public j: number,
-        public value: T
+        public value: T,
     ) {}
 
+    // this looks like a bug in the rule...
```

Comment:
instead of speculating in code comments, can you please file the issue and link to it? :)